.gitlab-ci.yml

```diff
@@ -168,6 +168,17 @@ cuda-maxset:
     - linux
     - cuda
 
+cuda-tutorials:
```

Comment:
why is the job named cuda-tutorials? we are not only checking cuda-tutorials but tutorials plus samples, right? Maybe just name it ```tutorials-samples```?

Reply:
Good idea.